portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,23 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['I can juggle', 'I know how to unicycle', 'I did the triple jump in high school', 'My major is Civil Engineering'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  var imgs = new Array ("images/juggling.png", "images/unicycle.png", "images/triplejump.png", "images/civil.png");
+
+  // Pick a random fact.
+  const index = Math.floor(Math.random() * facts.length);
+  const fact = facts[index];
+  const image = imgs[index];
```

Comment:
Here it would be a good idea to have a single array of (fact, image) tuples. It avoids any potential issues if the elements in both arrays ever get out of sync.

Something like..

```
const facts = [
  {
    text: 'I can pet ten dogs at a time',
    imageSrc: 'images/manydogs.png'
  },
  {
    ...
  }];
```

Reply:
That's a good idea, thank you, I just have a question before I make the change. I changed some of the facts/images in the random facts in a later pull request. If I change the format here will that mess up the later pull request? Or should I make the change at the later one?

Reply:
By making the change here, you will get merge conflicts with your later changes. In spite of this, I think the right thing to do would still be to make the change here and deal with the merge conflict.

What will happen is this:
1. You make the change on this branch and update the pull request
2. The pull request is accepted and merged into master
3. You will have to either merge the master branch into your other pending pull request or rebase the branch onto master (we can talk about this in more detail when we get there)
4. During the merge/rebase process, git will ask you to resolve the merge conflict
5. You go into your source file and fix the merge conflict by hand (it's fairly simple, I promise)
6. Push your merged/rebased branch up to Github to have an up to date pull request

Reply:
Okay, thank you. I just edited the file and committed it, so I'll update the pull request now.